app/views/admin/review_reminders/_form.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= form_with model: review_reminder, url: [:admin, document, review_reminder] do |form| %>
+  <%= render("components/datetime_fields", {
```

Comment:
There's def some duplication here with the other fields. I'm considering putting this into a view component.